core/src/main/scala/org/apache/spark/scheduler/LiveListenerBus.scala

```diff
@@ -226,3 +240,34 @@ private[spark] object LiveListenerBus {
   val name = "SparkListenerBus"
 }
 
+private[spark] class LiveListenerBusMetrics(queue: LinkedBlockingQueue[_]) extends Source {
+  override val sourceName: String = "LiveListenerBus"
+  override val metricRegistry: MetricRegistry = new MetricRegistry
+
+  /**
+   * The total number of events posted to the LiveListenerBus. This counts the number of times
+   * that `post()` is called, which might be less than the total number of events processed in
+   * case events are dropped.
```

Comment:
Yes. I was perhaps wasn't explicit enough in the comment, so I'll reword it or just drop the second confusing half.

Is it clearer if I say

> This counts the number of times that `post()` has been called called, not the total number of events that have completed processing.